Fix broken includes.

It looks like we've picked up some bit rot:

```
src/amimodel.cpp(17): error C2079: 'debug_stream' uses undefined class 'std::basic_ostringstream<char,std::char_traits<char>,std::allocator<char>>'
```

[This _Visual Studio_ community post](https://developercommunity.visualstudio.com/t/visual-studio-2019-compilation-issue/525532) suggests that we need to add:

```c++
#include <sstream>
```

to the `amimodel.cpp` file.

--- a/example/example_rx.cpp
+++ b/example/example_rx.cpp
@@ -7,6 +7,7 @@
  * Copyright (c) 2015 David Banas; all rights reserved World wide.
  */
 
+#include <sstream>
 #include <string>
 #include <vector>
 #include "include/ami_rx.h"

--- a/example/example_tx.cpp
+++ b/example/example_tx.cpp
@@ -10,6 +10,7 @@
 
 #define TAP_SCALE 0.0407
 
+#include <sstream>
 #include <string>
 #include <vector>
 #include "include/ami_tx.h"

--- a/include/amimodel.h
+++ b/include/amimodel.h
@@ -14,10 +14,10 @@
 
 #include <boost/spirit/include/qi.hpp>
 #include <boost/spirit/include/qi_omit.hpp>
-#include <boost/spirit/include/phoenix_core.hpp>
-#include <boost/spirit/include/phoenix_fusion.hpp>
-#include <boost/spirit/include/phoenix_operator.hpp>
-#include <boost/spirit/include/phoenix_stl.hpp>
+#include <boost/phoenix/core.hpp>
+#include <boost/phoenix/fusion.hpp>
+#include <boost/phoenix/operator.hpp>
+#include <boost/phoenix/stl/container.hpp>
 #include <boost/variant.hpp>
 #include <boost/fusion/include/adapt_struct.hpp>
 #include <fstream>

--- a/src/ami_rx.cpp
+++ b/src/ami_rx.cpp
@@ -7,6 +7,7 @@
  * Copyright (c) 2015 David Banas; all rights reserved World wide.
  */
 
+#include <sstream>
 #include <string>
 #include <vector>
 #include <algorithm>

--- a/src/ami_tx.cpp
+++ b/src/ami_tx.cpp
@@ -7,6 +7,7 @@
  * Copyright (c) 2015 David Banas; all rights reserved World wide.
  */
 
+#include <sstream>
 #include <string>
 #include "include/ami_tx.h"
 

--- a/src/amimodel.cpp
+++ b/src/amimodel.cpp
@@ -7,6 +7,7 @@
 * Copyright (c) 2015 David Banas; all rights reserved World wide.
 */
 
+#include <sstream>
 #include <string>
 #include <utility>
 #include <vector>

